### PR TITLE
Add owner_id to extractors

### PR DESCRIPTION
**Pull request by ccurme**
https://github.com/langchain-ai/langchain-extract/issues/43

Here we just add the column.

### CODE REVIEW

File: backend/server/api/extractors.py [hunk 1]
```diff
@@ -22,6 +22,10 @@ class CreateExtractor(BaseModel):
 
     name: str = Field(default="", description="The name of the extractor.")
 
+    owner_id: UUID = Field(
```
Comment (eyurtsev) [inline backend/server/api/extractors.py:25]: We want this information to come from the request header

File: backend/server/extraction_runnable.py [hunk 2]
```diff
@@ -188,7 +188,7 @@ async def extract_entire_document(
     text_splitter = TokenTextSplitter(
         chunk_size=get_chunk_size(model_name),
         chunk_overlap=20,
-        model_name=model_name,
+        model_name=DEFAULT_MODEL,
```
Comment (ccurme) [inline backend/server/extraction_runnable.py:191]: unrelated bug. for now we'll use gpt 3.5 tokenizer everywhere.

File: backend/db/models.py [hunk 3]
```diff
@@ -168,3 +173,16 @@ class Extractor(TimestampedModel):
...
     def __repr__(self) -> str:
         return f"<Extractor(id={self.uuid}, description={self.description})>"
+
+
+def validate_extractor_owner(
+    session: Session, extractor_id: UUID, owner_id: UUID
+) -> Extractor:
+    """Validate the extractor id."""
+    extractor = (
+        session.query(Extractor).filter_by(uuid=extractor_id, owner_id=owner_id).first()
```
Comment (eyurtsev) [inline backend/db/models.py:183]: .first() -> should usually be .scalar() or .one() if there are unique constraints

File: backend/server/api/examples.py [hunk 4]
```diff
@@ -55,8 +59,11 @@ def list(
     limit: int = 10,
     offset: int = 0,
     session=Depends(get_session),
+    owner_id: UUID = Cookie(...),
 ) -> List[Any]:
     """Endpoint to get all examples."""
+    if not validate_extractor_owner(session, extractor_id, owner_id):
```
Comment (eyurtsev) [inline backend/server/api/examples.py:65]: This works

**Review by eyurtsev — APPROVED** [review]

**Comment by eyurtsev** [comment]
Looks great! Merge whenever

**Pull request merged**